h2o-webserver-iface/src/main/java/water/webserver/iface/HttpServerFacade.java

```diff
@@ -0,0 +1,24 @@
+package water.webserver.iface;
+
+/**
+ * Facade for servlet container. We typically use Jetty behind this; however, due to use of various major versions,
```

Comment:
The word servlet container is misleading. There could be anything inside really.